### ISSUE TITLE
Run btc-rpc-explorer outside of the root domain

### ISSUE THREAD

**Issue by lukechilds** [janoside/btc-rpc-explorer#252]
Is it possible to run btc-rpc-explorer outside of the root domain?

It looks like images/URLs are hardcoded to use the root domain. If I expose the server on a path via a proxy the images cannot be loaded and links are broken. Is there a way to support this?

e.g If I run btc-rpc-explorer at http://umbrel-dev.local everything works. But if I run it at http://umbrel-dev.local/app/btc-rpc-explorer/ no images can be loaded and all links break.

<img width="1552" alt="Screenshot 2020-11-20 at 19 10 40" src="https://user-images.githubusercontent.com/2123375/99799920-10983080-2b66-11eb-9954-ae34d8e6eda1.png">

Is there a way to set an alternate site root?

**Comment by shesek** [comment]
Related: #207 #205

**Comment by Kixunil** [comment]
I use #207 but it really needs merging. Also the subpath is still fixed, so that needs to be solved. Unfortunately @janoside seems to be ignoring various PRs without any explanation, which doesn't help moving this project forward, so I'm seriously considering forking it.

**Comment by Kixunil** [comment]
So I made an updated version of baseurl PR. You may find it useful. I also plan to implement a few other features.

**Comment by janoside** [comment]
@Kixunil Thanks for the updated PR and the kick in the pants on this issue. I've taken a look in the past, and certainly support the concept, but was wishful that there was a different approach that wouldn't affect as much code. Each time I looked I failed to find a better solution but ate up all my free time looking rather than testing/merging. I agree this has sat too long and I'll focus on getting it merged and included in a release.
